content/es/docs/demo/_index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Documentación de la demo de OpenTelemetry
+linkTitle: Demo
+cascade:
+  repo: https://github.com/open-telemetry/opentelemetry-demo
+weight: 180
+cSpell:ignore: OLJCESPC
+---
+
+Bienvenido a la documentación de la [Demo de OpenTelemetry](/ecosystem/demo/), que
+cubre cómo instalar y ejecutar la demostración, y algunos escenarios que puedes usar para ver
+OpenTelemetry en acción.
+
+## Ejecución de la Demo
```

Comment:
```suggestion
## Ejecución de la demo
```

Title case is not a thing in Spanish, so all titles we found in EN that use title case should be converted to sentence case.